### DISCUSSION TITLE
How to get copilot for free?

### DISCUSSION THREAD

**Discussion by edm2k** [Schweinepriester/github-profile-achievements#328] [Q&A] [answered]
How to get copilot for free?

**Comment by Saidur07** [comment]
Apply to the Github Students pack. It's the easiest way to get copilot for free forever. Here you can apply for this and get so many staffs for free. 

https://education.github.com/pack

**Comment by devEyosiyas** [comment, marked as answer]
GitHub will offer Copilot for free to verified students and to maintainers of popular open source projects.
